buyer-api/src/services/batchInfo.js

```diff
@@ -0,0 +1,64 @@
+import { createLevelStore, listLevelKeys } from '../utils';
+import config from '../../config';
+
+const ordersPerBatch =
+  createLevelStore(`${config.levelDirectory}/orders_per_batch`);
+
+/**
+ * @async
+ * @function listBatchIds
+ * @returns {Promise} Promise which resolves to a list of batch Ids.
+ */
+const listBatchIds = async () => listLevelKeys(ordersPerBatch);
+
+/**
+ * @function createBatchId
+ * @returns {Integer} A new Id for the batch of orders
+ */
+// eslint-disable-next-line
+const createBatch = async (payload = []) => {
+  const id = (new Date()).getTime();
```

Comment:
Use an UUID as the id in order to not generate any confusion (timestamps are very similar)

```js
const uuid = require('uuid/v4');
const id = uuid();
```

Reply:
I was between uuid/guid and timestamps, with no preferences, I'll change it 👍